lib/sedna/common/file_ops.py

```diff
@@ -287,21 +331,24 @@ def _download_s3(cls, client, uri, out_dir):
         bucket_name = bucket_args[0]
         bucket_path = len(bucket_args) > 1 and bucket_args[1] or ""
 
-        objects = client.list_objects(bucket_name,
-                                      prefix=bucket_path,
-                                      recursive=True,
-                                      use_api_v1=True)
+        objects = list(client.list_objects(bucket_name,
+                                           prefix=bucket_path,
+                                           recursive=True,
+                                           use_api_v1=True))
         count = 0
-
+        num = len(objects)
```

Comment:
Variable names of "num" and "count" seem to be not explicit enough. There is suggestion that "num" should be replaced by "obj_num". It also confuses me what variable "count" means as it is slightly difficult for me to understand its role at the first sight.